Rakefile

```diff
@@ -1,9 +1,29 @@
-# Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
-
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
 
-ZIP_URL = "https://github.com/projectblacklight/blacklight-jetty/archive/v4.10.3.zip"
+BLACKLIGHT_JETTY_VERSION = '4.10.4'
+ZIP_URL = "https://github.com/projectblacklight/blacklight-jetty/archive/v#{BLACKLIGHT_JETTY_VERSION}.zip"
+
 require 'jettywrapper'
+require 'rspec/core/rake_task'
+
+RSpec::Core::RakeTask.new(:spec)
+
+desc 'Execute the test build that runs on travis'
+task ci: [:environment] do
+  if Rails.env.test?
+    Rake::Task['db:migrate'].invoke
+    Rake::Task['pulmap:download_and_unzip_jetty'].invoke
+    Rake::Task['pulmap:configure_jetty'].invoke
+    jetty_params = Jettywrapper.load_config
+    Jettywrapper.wrap(jetty_params) do
+      Rake::Task['geoblacklight:solr:seed'].invoke
+
+      # run the tests
+      Rake::Task['spec'].invoke
```

Comment:
Line 24 still needs to be wrapped in a `Jettywrapper.wrap(Jettywrapper.load_config) do` block, but it's probably better to maintain the pulmap:setup tasks in one spot.